eventnotifier/http/http.go

```diff
@@ -167,21 +175,51 @@ func (n *Notifier) Notify(dis v1beta1.Disruption, event corev1.Event, notifType
 		req.Header.Add(headerKey, headerValue)
 	}
 
-	res, err := n.client.Do(req)
+	if n.authTokenProvider != nil {
+		token, err := n.authTokenProvider.AuthToken(context.Background())
+		if err != nil {
+			return fmt.Errorf("http notifier: unable to retrieve auth token through helper: %w", err)
+		}
+
+		req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", token))
+	}
 
+	res, err := n.client.Do(req)
 	if err != nil {
 		return fmt.Errorf("http notifier: error when sending notification: %w", err)
 	}
 
-	n.logger.Debugw("notifier: sending notifier event to http", "disruption", dis.Name, "eventType", event.Type, "message", notif.EventMessage)
+	n.logger.Debugw("http notifier: sending notifier event to http", "disruption", dis.Name, "eventType", event.Type, "message", notif.EventMessage)
```

Comment:
Let's move this debug log _before_ we send the message, that way we get this info before we exit on error